.github/workflows/test-suite.yml

```diff
@@ -0,0 +1,51 @@
+name: test-suite
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        package:
+        - test-suite-rust-numbers-wasm32-unknown-unknown
+        - test-suite-rust-numbers-wasm32-wasi
```

Comment:
Each test should only be built once. If the test involves WASI, it might need to be built with it but for now everything should build without it.

Reply:
Could you expand here a bit? Do you want me to just drop the WASI builds?

Do you mean you'd prefer to have a single package per-language?